web/vocab/TERMS.md

```diff
@@ -0,0 +1,109 @@
+
+## ExperienceEvent
+- A particular Event recorded by a Person to describe an experience they had and how it connects to religion in the area.  Aggregates the following concepts:
+     - Sense
+     - Practice
+     - Object
+     - Person
+     - Researcher
+     - Place
+
+## Event
+- A place and time at which some action was observed or is known to have happened.  See http://schema.org/Event
```

Comment:
Yeah, I am not exactly sure how to properly set that up between terms and context.  We can look into it a little deeper to improve it in a later iteration.  `Experience` is not a unit data type, it is a jumble of other unit data type concepts not encompassed by http://schema.org/Event on its own because of `RelatedObjects`, `RelatedSenses`, `RelatedPractices` and `Tags`.

Reply:
`Experience` is a data type, right? It extends `Event`. It inherits all the base properties (can have a time and place and such) and extends it with further decoration possible (`RelatedObjects`, etc). You have it all set up perfectly, I think. We just need to get our minds into the abstraction that it represents.

Reply:
It extends a http://schema.org/Event simply to allow a way to define RelatedObjects, RelatedSenses, RelatedPractices and Tags as descriptors of an Event so we can have a lived religion specific Experience. Experience = _collection of entities, senses and behaviors observed_ at a time and place.  Event = _happening_ at a time and place.  

Experience = Event when and only when there are no entities, senses and behaviors observed, all of which are Lived Religion context specific things.

So it occurs to me that we are saying there are lived religion specific properties that extend Event with our own ontology.  This can be described in context.json like 

```json
"Event":{
         "@id":"schema:Event",
         "@type":"@id",
         "@context":{
            "RelatedSenses":{
               "@id":"lr:RelatedSenses",
               "@type":"@id",
               "@container":"@set"
            },
            "RelatedPractices":{
               "@id":"lr:RelatedPractices",
               "@type":"@id",
               "@container":"@set"
            },
            "RelatedObjects":{
               "@id":"skos:hasArtefactRelatedPhysicalResource",
               "@type":"@id",
               "@container":"@set"
            }
          }
      }
```

We can separate events between each other with something like https://schema.org/disambiguatingDescription or https://schema.org/name.